src/index.js

```diff
@@ -31,17 +30,20 @@ class PubsubPeerDiscovery extends Emittery {
    * @constructor
    * @param {Libp2p} param0.libp2p Our libp2p node
    * @param {number} [param0.interval = 10000] How often (ms) we should broadcast our infos
+   * @param {Array<Multiaddr>} param0.multiaddrs Multiaddrs used by the node to listen.
```

Comment:
How are these intended to get passed in? I would recommend not doing this and instead we should just leverage the announce addrs. I don't believe that's done yet, so in the interim we may want to just do `this.libp2p.transportManager.getAddrs()`. This removes any need to add special options for the addresses, which is not friendly for new users.

Reply:
Ok, we can use the `transportManager.getAddrs()` and I will open an issue to use the `addressManager` in the future

Reply:
Removed the param, using the transport manager for now and issue created #6